gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-http-get/src/test/java/io/gravitee/plugin/entrypoint/http/get/HttpGetEntrypointConnectorTest.java

```diff
@@ -294,18 +297,26 @@ void shouldJsonArrayOfMessageAndCompleteAndEndWhenResponseMessagesComplete(boole
 
         assertThat(httpHeaders).isNotNull();
         assertThat(httpHeaders.get(HttpHeaderNames.CONTENT_TYPE)).isEqualTo(MediaType.APPLICATION_JSON);
-        verify(response).chunks(chunksCaptor.capture());
 
+        ArgumentCaptor<Flowable<Buffer>> chunksCaptor = ArgumentCaptor.forClass(Flowable.class);
+        verify(response).chunks(chunksCaptor.capture());
         final TestSubscriber<Buffer> chunkObs = chunksCaptor.getValue().test();
 
         chunkObs
             .await()
             .assertComplete()
             .assertValueCount(6)
-            .assertValueAt(0, message -> message.toString().equals("{\"items\":["))
+            .assertValueAt(
+                0,
+                message -> {
+                    logger.debug("message[0]: {}", message.toString());
```

Comment:
suggestion: remove debug logs

Reply:
It's done on purpose, in case the fix is not enough and the flaky reappears.